FeedbackService.UnitTests/Handlers/GetNewsTickerHandlerTests.cs

```diff
@@ -0,0 +1,86 @@
+﻿using FeedbackService.Core.Domains;
+using FeedbackService.Core.Interfaces.Repositories;
+using FeedbackService.Handlers;
+using HelpMyStreet.Contracts;
+using HelpMyStreet.Contracts.FeedbackService.Request;
+using HelpMyStreet.Utils.Enums;
+using HelpMyStreet.Utils.EqualityComparers;
+using HelpMyStreet.Utils.Models;
+using Moq;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace FeedbackService.UnitTests.Handlers
+{
+    public class GetNewsTickerHandlerTests
+    {
+        private Mock<IRepository> _repository;
+        private GetNewsTickerHandler _classUnderTest;
+        private List<FeedbackRatingCount> _feedbackRatingCount;
+        private IEqualityComparer<NewsTickerMessage> _equalityComparer;
+
+        [SetUp]
+        public void Setup()
+        {
+            SetupRepository();
+            _equalityComparer = new NewsTickerMessages_EqualityComparer();
+            _classUnderTest = new GetNewsTickerHandler(_repository.Object);
+        }
+
+        private void SetupRepository()
+        {
+            _repository = new Mock<IRepository>();
+            _repository.Setup(x => x.FeedbackSummary(It.IsAny<int?>()))
+                .ReturnsAsync(() => _feedbackRatingCount);
+        }
+
+        [TestCase(RequestRoles.Volunteer, 100, 10, " from volunteers", 2)]
```

Comment:
I'd prefer to see the full message(s) specified here, rather than working them out below.